### DISCUSSION TITLE
[feature request] make it possible to copy links in reading mode

### DISCUSSION THREAD

**Discussion by Simplenuity** [gsantner/markor#1788] [Ideas] [answered]
Currently, if I'm in reading mode I can select plain text and am provided with a menu to Copy, Share, etc. it.
With any kind of link however, be it an URL or email address this doesn't work. Therefore my request to make it also possible for links. Ideally with the option to copy the text or the link itself (if it defers from the shown text).

Thank you for your consideration.

**Comment by gsantner** [comment]
= webviewclient context menu

**Reply by Simplenuity** [reply]
Thanks! So, sadly not under your control

**Reply by gsantner** [reply]
it is, but needs somebody to develop it.

**Comment by gsantner** [comment]
= webviewclient context menu

**Comment by brunetton** [comment]
Workaround: use [Link Eye](https://f-droid.org/packages/kuesji.link_eye.fdroid) and use its copy function for each opened URL

**Reply by Julrob199** [reply]
Using another browser or third app is not a workaround, it changes the whole other systems behaviour. What is a workaround: Select normal text and move the selector so it only selects the url link.

**Comment by Julrob199** [comment]
Would highly appreciate such feature

**Comment by guanglinn** [comment, marked as answer]
I just implemented this feature in PR #2730. It's simple but should be useful.

https://github.com/user-attachments/assets/a766b365-e711-4d32-97d0-4382a227b539

**Comment by guanglinn** [comment]
Improved this feature in PR #2798.

**Comment by ravilov** [comment]
That's quite good. Looking forward to this being merged in.

Maybe one day we'll have an actual rightclick-style popup menu. 🙂
